patterns/stories-podcast-hero.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Title: Stories Podcast Hero
+ * Slug: twentytwentyfive/stories-podcast-hero
+ * Categories: featured, hero
+ * Keywords: podcast, hero, stories
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
```

Comment:
done

Reply:
@iamsam2e You have removed wrong empty line. In Pattern Document Before `?>`  empty line is required, else it will give lint issue same as current)

You need to remove Empty Line before `<!-- wp-group -->` start as mentioned.

Reply:
Oh youre right sorry I did do the wrong one 🤦🏽 

pushing that fix now